### ISSUE TITLE
Questions about the half-implemented text chat feature

### ISSUE THREAD

**Issue by keratomalacian** [GlobedGD/globed2#263]
Hi, after taking a look at the server source code there seems to be some mentions about a text-based chat system. even with an already defined packet. But in the protocol.md file the feature appears to be marked as not fully functional.

Is this feature currently in development? Was it scrapped off in favor of the voice system? Or what happened to it?

**Comment by dankmeme01** [comment]
Internally, on the server side it is fully functional (and has been since release of the mod)

However on the client side we haven't gotten around implementing it yet, due to certain difficulties and also concerns of the staff team (text chat would be a lot more effort to moderate than voice chat)

**Comment by keratomalacian** [comment]
Wouldn't it be easier to moderate a text chat system than a voice chat system? I think there is already a list of swear words in the game that can't be used and reported messages could be sent to the staff for review

**Comment by dankmeme01** [comment]
The problem is volume, filters are fairly easy to bypass, and text chat would be used *a lot* more than voice chat, since it would work everywhere and for everyone, while voice chat can only be used on Windows and if you have a microphone.

Given that the main servers consistently have 2-3k players online everyday at peak times, I feel like it would just be too much work to moderate

**Comment by keratomalacian** [comment]
Yeah I guess that's fair. Thanks for the information, closing this now.
